domain/robstoll-lib/atrium-domain-robstoll-lib-common/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/charsequence/contains/creators/creators.kt

```diff
@@ -59,17 +66,22 @@ fun <T : CharSequence> _containsDefaultTranslationOfIgnoringCase(
 
 fun <T : CharSequence> _containsRegex(
     checkerOption: CharSequenceContains.CheckerOption<T, NoOpSearchBehaviour>,
-    expected: List<String>
+    expected: List<Regex>
 ): AssertionGroup = createAssertionGroup(checkerOption, RegexSearcher(), expected, STRING_MATCHING_REGEX)
 
 fun <T : CharSequence> _containsRegexIgnoringCase(
     checkerOption: CharSequenceContains.CheckerOption<T, IgnoringCaseSearchBehaviour>,
     expected: List<String>
-): AssertionGroup = createAssertionGroup(checkerOption, IgnoringCaseRegexSearcher(), expected, STRING_MATCHING_REGEX)
+): AssertionGroup = createAssertionGroup(
+    checkerOption,
+    IgnoringCaseRegexSearcher(),
+    expected.map { it.toRegex() },
```

Comment:
should not be necessary (revert the change)

Reply:
I think it fails to compile if I revert `expected.map { it.toRegex() }` to `expected`.

Reply:
you are right. Well, in this case I have an additional suggestion. It does not really make sense to convert it here to regex and then unpack it in `IgnoreCaseRegexSearcher` just to repack it with the RgexOptin.IGNORE_CASE. Thus:
- use the following here: `Regex(it, RegexOption.IGNORE_CASE)`
- use the RegexSearcher one line above
- remove IgnoringCaseRegexSearcher